### DISCUSSION TITLE
Response for proposed approach for #4774

### DISCUSSION THREAD

**Discussion by ghost** [tokio-rs/tokio#4889] [General] [answered]
I was wondering if I can get some response for the proposed approach for issue #4774 

https://github.com/tokio-rs/tokio/issues/4774#issuecomment-1206810196

**Comment by Noah-Kennedy** [comment, marked as answer]
Let's keep the discussion in that issue so that it's easier to follow.

For the record, I don't think anyone was ignoring you, it's just that a bunch of us were at rustconf the past few days ;)
